File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/text/ReplaceTextWorksheet.java
package com.aspose.cells.cloud.examples.text;

import java.io.IOException;
import java.nio.file.Path;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

public class ReplaceTextWorksheet {

    public static void main(String... args) throws IOException {
        // ExStart: replace-text-worksheet

        try {
            // Instantiate Aspose Storage API SDK
            StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

            // Instantiate Aspose Words API SDK
            CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
            String input = "sample1.xlsx";
            Path inputFile = Utils.getPath(ReplaceTextWorksheet.class, input);

            storageApi.PutCreate(input, null, Utils.STORAGE, inputFile.toFile());
            String oldValue = "aspose";
            String newValue = "aspose.com";
            String sheetName = "Sheet1";

            com.aspose.cells.model.WorksheetReplaceResponse apiResponse = cellsApi.PostWorsheetTextReplace(input,
                    sheetName, oldValue, newValue, Utils.STORAGE, Utils.FOLDER);

            if (apiResponse != null && apiResponse.getStatus().equals("OK")) {

                System.out.println("Matches: " + apiResponse.getMatches());
            }
        }

        catch (Exception e) {
            e.printStackTrace();
        }
        // ExEnd: replace-text-worksheet
    }

}

File: Examples/DotNet/CSharp/Text/ReplaceTextWorksheet.cs
﻿using System;
using Com.Aspose.Cells.Api;
using Com.Aspose.Cells.Model;
using Com.Aspose.Storage.Api;

namespace Text
{
    class ReplaceTextWorksheet
    {
        public static void Run()
        {
            // ExStart:1
            CellsApi cellsApi = new CellsApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            String fileName = "Sample_Test_Book.xls";
            String sheetName = "Sheet2";
            String oldValue = "aspose";
            String newValue = "aspose.com";
            String storage = null;
            String folder = null;

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));

                // Invoke Aspose.Cells Cloud SDK API to replace text in worksheet
                WorksheetReplaceResponse apiResponse = cellsApi.PostWorsheetTextReplace(fileName, sheetName, oldValue, newValue, storage, folder);

                if (apiResponse != null && apiResponse.Status.Equals("OK"))
                {
                    Console.WriteLine("Matches: " + apiResponse.Matches);
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}
